HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -268,6 +268,14 @@ def customizeHLTfor44576(process):
         break
     return process
 
+
+def customizeHLTfor45023(process):
+    for pset in process.psets.values():
+        if hasattr(pset, 'ComponentType') and pset.ComponentType == 'CkfTrajectoryBuilder':
+           pset.foundHitBonus = cms.double(0.0)
+           pset.minHitForDoubleBonus = cms.int32(9999)
+    return process     
```

Comment:
is the customization still necessary after the last round of updates? i.e. IIUC the current defaults in `fillDescriptions` will ensure there is no regression at HLT?

Reply:
In the old days tests would not pass as the HLT configs in the release do not contain those params.
If now they read from ```fillDescriptions``` the customize can of course be removed.

Reply:
>  If now they read from `fillDescriptions` the customize can of course be removed.

yes, it should be the case.